pyvista/plotting/themes.py

```diff
@@ -2914,6 +2918,28 @@ def lighting_params(self, config: _LightingConfig):  # numpydoc ignore=GL08
             raise TypeError('Configuration type must be `_LightingConfig`.')
         self._lighting_params = config
 
+    @property
+    def logo_file(self) -> Optional[pathlib.Path]:  # numpydoc ignore=RT01
+        """Return or set the logo file.
+
+        Examples
+        --------
+        >>> import pyvista as pv
+        >>> pv.global_theme.silhouette.opacity = 1.0
```

Comment:
Fix the example here.

Reply:
I opted to remove the example as the example would need to specify a path and wouldn't be copy/pastable for end users.

Reply:
Good point.  One thought is to make it clear that it is not copy/pastable like `pv.global_theme.logo_file = "path/to/image_file.txt". # Use your own path`.

Reply:
Done in 538533bb5dc722ed08f9329444a29a2ebd4eb7e9